lms/djangoapps/django_comment_client/utils.py

```diff
@@ -137,6 +138,33 @@ def get_accessible_discussion_modules(course, user, include_all=False):  # pylin
     ]
 
 
+def get_discussion_module(course, user, discussion_id):
+    """
+    Return discussion module in the course against a discussion id.
+
+    Parameters:
+        course_key: The course object.
+        user: The user requested the object.
+
+    Returns:
+        The discussion module information in the specified course for the given
+        discussion id only if user had access to that discussion module.
+
+    Raises:
+        DiscussionNotFoundError: if  discussion module does not exist for
+        given ID or requesting user does not have access to requested
+        discussion module.
+    """
+    key = get_cached_discussion_key(course, discussion_id)
+    if key:
+        discussion_module = modulestore().get_item(key)
+        if not discussion_category_id_access(course, user, discussion_id, discussion_module):
+            raise DiscussionNotFoundError("Discussion not found.")
+        return discussion_module
+    else:
+        raise DiscussionNotFoundError("Discussion not found.")
```

Comment:
The message included with the exception doesn't provide any useful information.  It just duplicates the name of the exception.  Either leave it out, or include the id of the discussion that was not found.